### ISSUE TITLE
Sitemap config checkbox not display correctly if on

### ISSUE THREAD

**Issue by bradleykronson** [patrickdemooij9/SeoToolkit.Umbraco#251]
On the sitemap config tab for document types, the Hide from sitemap checkbox is always showing off - it does not correctly display if it is on.

Seen in Umbraco 11 and 12 with the latest version of the toolkit

**Comment by chloeMoirae** [comment]
I am currently experiencing this same issue. Same as when I change the canonical, the new canonical doesnt show correctly in the sitemap

**Comment by patrickdemooij9** [comment]
I don't think this is an issue for the latest version of SeoToolkit anymore, hence I'll close the issue. Please reopen it if you still see this happening on the latest versions
